auto_route/lib/src/router/controller/routing_controller.dart

```diff
@@ -72,7 +72,7 @@ abstract class RoutingController with ChangeNotifier {
   RouteData _createRouteData(RouteMatch route, RouteData parent) {
     final mayUpdateController = childControllers[route.key];
 
-    var pendingChildren = const <RouteMatch>[];
```

Comment:
It's interesting to see what happens when you combing `var` with `const` in this case. You would think dart would have complained given that it's assigning an immutable object to a mutable var.